src/asmfiddle/machine/machine.go

```diff
@@ -352,6 +352,40 @@ func (s *stack) Pop() int {
 	return s.stack[s.sp]
 }
 
+
+// Utilities
+func Int2Bytes(int_val []int) []byte {
```

Comment:
This will also be used by jsui.go etc, so move it to asmfiddle package from machine package.